File: Python PowerPoint Examples/SlidesApiDemo/deleteSlideByIndex.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.slides_api.SlidesApi(configuration)

fileName = "DeleteSlide.pptx"
password = ""
storage = ""
slide_index = 10
inputFolder = "input"
result =api.delete_slide_by_index(fileName, slide_index=slide_index, password=password, storage=storage, folder=inputFolder)

File: PHP PowerPoint Examples/SlidesApiDemo/deleteSlideByIndex.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\SlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new SlidesApi($configuration);

$name = "DeleteSlide_1.pptx";
$password = null;
$folder = "/Slides/DeleteSlide/";
$storage = null;
$slideIndex = 0;
$apiInstance->deleteSlideByIndex($name, $slideIndex, $password, $folder, $storage);